packages/@sanity/default-login/src/CookieTest.tsx

```diff
@@ -0,0 +1,122 @@
+import React, {useEffect, useMemo, useState} from 'react'
+import {Container, Text, Flex, Spinner} from '@sanity/ui'
+import {fetchToken, saveToken} from '@sanity/base/_internal'
+import {versionedClient} from './versionedClient'
+import {LoginDialog, pluginConfig} from './legacyParts'
+
+const isProjectLogin = versionedClient.config().useProjectHostname
+const projectId: string | null =
+  (isProjectLogin && versionedClient.config && versionedClient.config().projectId) || null
```

Comment:
Seems like we don't support multiple projectId spaces because the authentication fetcher doesn't take this into account (it's run before spaces are configured)

Reply:
I have tried to solve this here: https://github.com/sanity-io/sanity/commit/a65f0fe117ac6fe09a28ff25cf890d9e91d07c62

But we need to check the URL what space is enabled as authentication fetcher is run before the `reconfigureClient` stuff is done for spaces in `default-layout`.

Reply:
(seems to me like we don't properly support multi project spaces in the current version either)